pyscriptjs/src/remote_interpreter.ts

```diff
@@ -72,9 +85,35 @@ export class RemoteInterpreter extends Object {
     // TODO: Remove this once `runtimes` is removed!
     interpreter: InterpreterInterface & ProxyMarked;
 
-    constructor(src = 'https://cdn.jsdelivr.net/pyodide/v0.23.2/full/pyodide.js') {
+    pyodide: Promise<PyodideHelper> | null;
+
+    constructor(src = 'https://cdn.jsdelivr.net/pyodide/v0.23.2/full/pyodide.js', bootstrap = false) {
         super();
         this.src = src;
+        this.pyodide = bootstrap
+            ? new Promise(resolve => {
```

Comment:
Do you think this code should be encapsulated into a function so we can do something like: `this.pyodide = bootstrap ? installPlugins()  : null` ? 

I don't really have an opinion here, just wondering 😄

Reply:
this code is dead already, as there's no plan to hook current core into "classic py-script" or unknown amount of shenanigans will happen (see CI complaining rightly so, there are things just different in core).

But also please read my previous comment which is about the dual meaning of plugins:

  1. one is the core plugin, like `customElements.define('selector', ...)` could be
  2. the other one is what `py-script`, as a plugin of core, export as `registerPlugin` definition